<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F1_SystemClass</class>
 <widget class="QMainWindow" name="F1_SystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>F1 Tournament Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">
	background-image: url(../backgroundLogin.jpg);</string>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>601</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
color: white;
</string>
    </property>
    <property name="text">
     <string>F1 Management System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>320</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: 0;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameInputField">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;
background: white;
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordInputField">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>60</y>
       <width>190</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid white;
background: white;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>95</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: red; color: white;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent; color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="f1_system.qrc"/>
 </resources>
 <connections/>
</ui>
